docs/userguides/development.md

```diff
@@ -7,22 +7,60 @@ In this guide, we are going to show you more details on how to build an applicat
 You should have a python project with Silverback installed.
 You can install Silverback via `pip install silverback`
 
+## Project structure
+
+The `silverback` cli automatically searches for python scripts to run as bots in specific locations relative to the root of your project.
+It if finds a module named `bot.py` either in the `bots/` directory or at the root directory of the project, then it will use that by default.
```

Comment:
I dont think `bots/bot.py` should be recommended, I think it's either `bot.py`, `bot/__init__.py` (aka a module named `bot` for both cases), or a unique named script under `bots/`

Reply:
This requires a bit of changing for the dockerfile build

Reply:
Also, what if we allow them to name that folder whatever they want. If each bot requires a complicated project structure, shouldn't we allow them to use all of these features? Should we do this in a follow-up PR?

Reply:
The only thing we need to do in the dockerfile is if they are using a script at path `bots/{name}.py` to move it to `bot.py` inside `Dockerfile.{name}` (and ignore the rest of the stuff in `bots/`)

Everything else is likely too complicated to be changing project structure around, and they should be making their own docker image manually

Reply:
I read through that paragraph and I'm not exactly suggesting to name your file `bot.py`. I'm stating that it automatically detects that file name. The rest of that section explains the different ways the project structure can be ran, do you want me to change that all together?

Reply:
I want to remove `bots/bot.py` as autodetected

Reply:
Made the change, let me know if that works